packages/itwinui-css/src/workflow-diagram/workflow-diagram.scss

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../stepper/index';
+
+@mixin iui-workflow-diagram {
+  @include iui-stepper;
+
+  > ol {
+    align-items: center;
+  }
+}
+
+@mixin iui-workflow-diagram-step {
+  @include iui-stepper-step;
+
+  --_iui-workflow-diagram-circle-background-color: var(--iui-color-background-positive);
+  --_iui-workflow-diagram-circle-border-color: var(--iui-color-background-positive);
+  --_iui-workflow-diagram-circle-border-radius: 0;
+  --_iui-workflow-diagram-circle-text-color: var(--iui-color-foreground-accessory);
+
+  color: var(--_iui-workflow-diagram-circle-text-color);
+
+  &:first-of-type,
+  &:last-of-type {
+    --_iui-workflow-diagram-circle-background-color: var(--iui-color-background-1);
+    --_iui-workflow-diagram-circle-border-color: var(--iui-color-foreground-positive);
+    --_iui-workflow-diagram-circle-border-radius: #{$iui-sm};
+    --_iui-workflow-diagram-circle-text-color: var(--iui-color-foreground-positive);
+  }
+}
+
+@mixin iui-workflow-diagram-circle {
```

Comment:
This could be renamed since they are not circles, unsure what to name them.

Reply:
`iui-workflow-diagram-item` ?

Reply:
Or perhaps change `-content` to `-item` and then change `-circle` to `-content`?